### DISCUSSION TITLE
Decorating with External Data

### DISCUSSION THREAD

**Discussion by eecue** [RhetTbull/osxphotos#908] [Q&A] [answered]
I am interested in using AWS Rekognize Celebrity detection to tag celebs (mostly musicians) and then turn those into People in Photos.  Is that possible?

**Comment by RhetTbull** [comment, marked as answer]
It would be possible to add keywords associated with the celeb name or stick them in a folder with the person's name. It is not currently possible to tag persons programmatically. I am working on adding this capability to osxphotos for merging libraries (#356) but it involves some pretty deep hacking of the Photos database and it's not something that will be done soon.  It's more of a back burner project I work on when time allows.

**Reply by eecue** [reply]
Yep that makes sense, I was already planning on adding the celeb names as tags.  Thanks!

**Reply by eecue** [reply]
Thinking about this more, it’s something i’d like to contribute. Photos does a fairly good job of finding faces but a pretty bad job of giving you a way to manage the unlabeled ones.

**Comment by SuelenMaria** [comment]
No problem, thanks!

Em qui., 19 de jan. de 2023 13:49, Rhet Turnbull ***@***.***>
escreveu:
